src/NServiceBus.Transport.Msmq.Tests/InstanceMapping/InstanceMappingFileFeatureTests.cs

```diff
@@ -11,13 +11,15 @@ public class InstanceMappingFileFeatureTests
         [Test]
         public void Should_configure_default_values()
         {
-            var feature = new InstanceMappingFileFeature();
-            var settings = new SettingsHolder();
+            // Test has to be commented out for now, no way to access the defaults
```

Comment:
Should be possible to unblock this by making `registeredDefaults` protected, assuming we're OK with giving features access to the collection.